includes/admin/class-wc-rest-payments-orders-controller.php

```diff
@@ -76,6 +76,20 @@ public function register_routes() {
 				],
 			]
 		);
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/(?P<order_id>\w+)/capture_authorization',
```

Comment:
Also, it'd be great to have it documented under https://github.com/Automattic/woocommerce-payments/blob/develop/docs/rest-api/source/includes/wp-api-v3/order.md